Detectors/TPC/reconstruction/run/testRawReader.cxx

```diff
@@ -0,0 +1,95 @@
+///
+/// @file   testRawReader.cxx
```

Comment:
we could consider making a "prototyping" directory. In any case, we should still try to implement proper unit tests for classes you develop.